src/components/OpenWith/OpenWithDropdownMenuItem.js

```diff
@@ -15,12 +15,12 @@ type Props = {
 };
 
 const OpenWithDropdownMenuItem = ({
-    integration: { displayName, displayDescription },
+    integration: { displayName, displayDescription, appIntegrationId },
     onClick,
 }: Props) => {
     const Icon = ICON_FILE_MAP[displayName] || IconFileDefault;
```

Comment:
I think this is being repeated.
Probably have a function inside IconFileMap.js and export that instead.
getIcon()

Reply:
https://github.com/box/box-ui-elements/pull/564/files#diff-bb9a336e6b9e61655031483d8276ded2R64

Reply:
Yeah Jared pointed this out as well. It's a good idea but I need to be able to distinguish if It's the default icon or not for style purpose. I could do an instance of check in the component...